plugins/catalog/src/components/EntityContextMenu/EntityContextMenu.tsx

```diff
@@ -84,6 +86,14 @@ export function EntityContextMenu(props: EntityContextMenuProps) {
     setAnchorEl(undefined);
   };
 
+  const alertApi = useApi(alertApiRef);
+
+  const copyToClipboard = useCallback(() => {
+    navigator.clipboard
+      .writeText(window.location.toString())
+      .then(() => alertApi.post({ message: 'Copied!', severity: 'info' }));
+  }, []);
+
```

Comment:
warning: "React Hook useCallback has a missing dependency: 'alertApi'. Either include it or remove the dependency array" 
Shall I include the **alertApi** in the dependency array?

`      .then(() => alertApi.post({ message: 'Copied!', severity: 'info' }));
  }, [alertApi]); `

Reply:
Yes.